bmc-acf/acf_manager.cpp

```diff
@@ -109,20 +108,19 @@ std::tuple<std::vector<uint8_t>, bool, std::string> ACFCertMgr::getACFInfo(void)
 
     if (!readBinaryFile(ACF_FILE_PATH, accessControlFile))
     {
-        log<level::ERR>("ACF not installed or not readable");
+        lg2::error("ACF not installed or not readable");
     }
     else
     {
         // Verify ACF and get expiration date.
         Tacf tacf{[](std::string msg) {
-            log<phosphor::logging::level::INFO>(msg.c_str());
+            lg2::info(msg.c_str());
         }};
         int rc = tacf.verify(accessControlFile.data(), accessControlFile.size(),
                              sDate);
         if (rc)
         {
-            log<level::INFO>("ACF is not valid");
-            log<level::ERR>("Error: ", entry("rc=%0x", rc));
+            lg2::error("Error: ACF is not valid, rc=0x{RC}", "RC", lg2::hex, rc);
```

Comment:
lg2 will add the 0x itself so you don't need to.

Reply:
Thanks!  I've removed it now.